pkg/apis/litmuschaos/v1alpha1/chaosengine_types.go

```diff
@@ -30,6 +30,9 @@ type ChaosEngineSpec struct {
 	Appinfo ApplicationParams `json:"appinfo,omitempty"`
 	//AnnotationCheck defines whether annotation check is allowed or not. It can be true or false
 	AnnotationCheck string `json:"annotationCheck,omitempty"`
+	//DefaultChecks defines whether default health checks should be executed or not. It can be true or false
+	// default value is true
+	DefaultChecks string `json:"defaultChecks,omitempty"`
```

Comment:
should we make this key more descriptive? Such as DefaultAppHealthCheck ? (for nodes it is better to do)